graylog2-server/src/main/java/org/graylog2/alerts/types/MessageCountAlertCondition.java

```diff
@@ -26,28 +26,46 @@
 import org.graylog2.indexer.results.SearchResult;
 import org.graylog2.indexer.searches.Searches;
 import org.graylog2.indexer.searches.Sorting;
+import org.graylog2.plugin.Message;
+import org.graylog2.plugin.MessageSummary;
+import org.graylog2.plugin.Tools;
 import org.graylog2.plugin.alarms.AlertCondition;
+import org.graylog2.plugin.configuration.ConfigurationRequest;
+import org.graylog2.plugin.configuration.fields.ConfigurationField;
+import org.graylog2.plugin.configuration.fields.DropdownField;
+import org.graylog2.plugin.configuration.fields.NumberField;
 import org.graylog2.plugin.indexer.searches.timeranges.AbsoluteRange;
 import org.graylog2.plugin.indexer.searches.timeranges.InvalidRangeParametersException;
 import org.graylog2.plugin.indexer.searches.timeranges.RelativeRange;
-import org.graylog2.plugin.Message;
-import org.graylog2.plugin.MessageSummary;
-import org.graylog2.plugin.Tools;
 import org.graylog2.plugin.streams.Stream;
 import org.joda.time.DateTime;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import javax.annotation.Nullable;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Locale;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 public class MessageCountAlertCondition extends AbstractAlertCondition {
     private static final Logger LOG = LoggerFactory.getLogger(MessageCountAlertCondition.class);
 
-    public enum ThresholdType {
-        MORE, LESS
+    enum ThresholdType {
+
+        MORE("more than"),
+        LESS("less than");
```

Comment:
Should there also be an `EXACT` threshold type (e. g. exactly 10 messages) or is this subject to a future feature (if at all)?